components/tidb_query/src/codec/mysql/decimal.rs

```diff
@@ -1793,20 +1794,13 @@ impl ConvertTo<Decimal> for Real {
 }
 
 impl ConvertTo<Decimal> for &[u8] {
+    // FIXME, the err handle is not exactly same as TiDB's,
+    //  TiDB's seems has bug, fix this after fix TiDB's
```

Comment:
```suggestion
    // TiDB's seems has bug, fix this after fix TiDB's
```

Reply:
Add this space to indicate this is belong to the section of `FIXME`, and in CLion, it will look like this
![image](https://user-images.githubusercontent.com/21231345/64674370-48581280-d4a3-11e9-9221-3b4dfc4d6571.png)